Added spotless java formatter plugin and updated readme (#105)
Real-Dev-Squad/skill-tree-backend: README.md

10. Click the green "Run" button or "Shift + F10" to start the application
11. After starting the Tomcat server on port `8080`, attempt to access the dummy route `http://localhost:8080/test` using the `GET` method in Postman or ThunderClient while providing the `bearer token`. If the terminal displays `test123`, it indicates that the setup has been successful.

## Contributing
### Code Formatting

This repo uses https://github.com/diffplug/spotless/tree/main/plugin-maven#java for formatting files.

> Please run `mvn spotless:apply` before check-in to fix any formatting errors.

Reviewer: check-in means?

Author: By check-in, I meant pushing the code in the repository. I'll make it more clear. Thank you

> The Continuous Integration build for pushed commits may fail when a Pull Request is created if formatting is not applied.

Reviewer: ```suggestion
The Continuous Integration build for pushed commits may fail when a Pull Request is created if your code doesn't follow project's formatting guideline.
```
